pyomo/solvers/plugins/solvers/persistent_solver.py

```diff
@@ -377,12 +377,16 @@ def solve(self, *args, **kwds):
 
             # we're good to go.
             initial_time = time.time()
+            initial_det_time = self._solver_model.get_dettime()
```

Comment:
Unfortunately I don't think this will work...

`get_dettime()` is only exposed by the CPLEX solver API however this class is intended to be general for all possible PersistentSolvers (eg. Gurobi) as well which won't have this same interface.